src/web/components/Branding.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { LinkStyle } from '@root/src/amp/components/elements/TextBlockComponent';
+import { textSans } from '@guardian/src-foundations/typography';
+
+import { css } from 'emotion';
+import { palette } from '@guardian/src-foundations';
```

Comment:
We recently moved away from importing the whole `palette` object in favour of just the thing you need. Eg:

`import { neutral} from '@guardian/src-foundations/palette';`